src/main/java/org/openbase/bco/bcozy/controller/powerterminal/chartattributes/Granularity.java

```diff
@@ -0,0 +1,11 @@
+package org.openbase.bco.bcozy.controller.powerterminal.chartattributes;
+
+public enum Granularity {
+    CONSUMER, ROOM, OVERALL;
+
+
+    @Override
+    public String toString() {
+        return super.toString().substring(0, 1) + super.toString().substring(1).toLowerCase();
```

Comment:
You can use `StringProcessor.transformFirstCharToUpperCase(super.name().toLowerCase())` for something like this. But in general please use `LanguageSelection.getLocalized(myEnum.name())` to use the enum name for any UI components.

Reply:
Resolved in 62de25f